docker-compose.yml

```diff
@@ -81,6 +81,7 @@ services:
       - NODE_ENV=testing
       - SKIP_SIGNATURE_VERIFICATION=true
       - ALLOW_PRIVATE_ADDRESS=true
+      - NODE_TLS_REJECT_UNAUTHORIZED=0
```

Comment:
@allouis I needed to add this otherwise the webfinger lookup on the `fake-external-activitypub` would not work.

When Fedify does the lookup based on just being provided a handle (`@mike@mikes.world`) it defaults to using `https` so i had to expose the `--https-port` on the wiremock containers. This uses a self signed certificate, so had to enable `NODE_TLS_REJECT_UNAUTHORIZED` so that node didn't complain about the certificate.

 Not sure if i've not set something up correct somewher thats meaning i need to do this?